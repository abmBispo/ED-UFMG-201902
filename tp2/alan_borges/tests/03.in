1000 3 1
500 a
1000 b
500 c