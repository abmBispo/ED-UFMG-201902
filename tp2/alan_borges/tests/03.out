1 a 500
1 c 500
2 b 1000
